pkg/internal/testabilities/testservices/fixture_bhs.go

```diff
@@ -114,5 +124,17 @@ func newDefaultLongestChainTipResponse() *longestChainTipResponse {
 	}
 }
 
+const tipLongestPath = "/api/v1/chain/tip/longest"
+
 var bhsTestURLWithoutHTTPprefix = defs.BHSTestURL[7:]
-var bhsAnyEndpointRegexFixture = regexp.MustCompile(fmt.Sprintf(`^http:\/\/%s(?:\/.*)?$`, bhsTestURLWithoutHTTPprefix))
+var bhsAnyEndpointRegexFixture = regexp.MustCompile(fmt.Sprintf(`^http:\/\/%s%s$`, regexp.QuoteMeta(bhsTestURLWithoutHTTPprefix), tipLongestPath))
+
+func (f *bhsFixture) HttpClient() *resty.Client {
```

Comment:
"Receiver names are different"

```suggestion
func (b *bhsFixture) HttpClient() *resty.Client {
```